pkgs/development/python-modules/phik/default.nix

```diff
@@ -50,8 +52,11 @@ buildPythonPackage rec {
   nativeBuildInputs = [
     cmake
     ninja
+    scikit-build
   ];
 
+  pythonImportCheck = [ "phik" ];
+
   postInstall = ''
   rm -r $out/bin
```

Comment:
```suggestion
    rm -r $out/bin
```

Reply:
Can do, but this is how it was before me. I try to make PR as small as possible.